### PR TITLE
Add a test game

### PR DESCRIPTION
**Pull request by jacklehamster**

### CODE REVIEW

**Comment by leobenkel** [comment]
Do you want to make the game into its own repo ? So the library stay light ? You would just have to copy the `demo.html` and `demo.js` and `js/main-demo.js` . If you want to add this game as an extra demo then can you move the file into `game-files-examples` ? And add a html / js to be able to have a URL to hit to test it ?

**Comment by leobenkel** [comment]
I went all the way to the temple after taking the shell and the monkey and the crab, what is happening in the game ?! I want to know the end of the mystery !

File: game/game.json [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  },
+  "scenes": {
+    "beach": {
+      "name": "beach",
+      "backgroundImg": "https://ak.picdn.net/shutterstock/videos/2763035/thumb/1.jpg",
+      "interactions": [
+        {
+          "actions": [
+            "takeCrab",
+            {
```
Comment (leobenkel) [inline game/game.json:36]: you can more this action inside the global action `takeCrab` so everytime the action of taking the crab is executed , the `hasCrab` variable is set to `true`

**Review by leobenkel — CHANGES_REQUESTED** [review]
Do you want to make the game into its own repo ? So the library stay light ? You would just have to copy the demo.html and demo.js and js/main-demo.js . If you want to add this game as an extra demo then can you move the file into game-files-examples ? And add a html / js to be able to have a URL to hit to test it ?

**Comment by jacklehamster** [comment]
How can I import the library to a new repo, can I just lind directly into https://the-brains.github.io/ClickAndPointLib/...?

**Comment by leobenkel** [comment]
yeah, like we did for the other libraries. There is the dist folder you can just import: https://the-brains.github.io/ClickAndPointLib/dist/clickAndPoint.js

**Comment by jacklehamster** [comment]
Closing. I kinda forgot what this was all about.

**Pull request closed without merging**